features/support/http.rb

```diff
@@ -0,0 +1,9 @@
+module HttpHelpers
+  def http_delete(path)
```

Comment:
Switching the driver [will not retain session or cookie information from previous requests](https://stackoverflow.com/questions/9228831/how-to-make-capybara-do-a-delete-request-in-a-cucumber-feature#comment60760638_19244681), so you're not logging out as the user you want logged out.

An alternative we could try is to [keep the JS driver and make the DELETE request using jQuery](https://stackoverflow.com/a/36580783). We already have a function for the waiting part:

https://github.com/otwcode/otwarchive/blob/df7cac7f4103b8fa1a42cfe509097ad4cb71e9e2/features/support/wait_for_ajax.rb#L3-L7

Can you add to the existing file `features/support/capybara.rb` instead of adding `features/support/http.rb`?

Reply:
This causes the following error in tests:

    undefined method `zero?' for nil:NilClass (NoMethodError)

    ./features/support/wait_for_ajax.rb:10:in `finished_all_ajax_requests?'

I'm not sure if this is an issue with my Vagrant install (PhantomJS initially didn't work at all, I had to run `export QT_QPA_PLATFORM=offscreen` before anything else), or if it's something that actually broke.

Reply:
Done, but the Ajax request fails and throws this error now:

    ReferenceError: Can't find variable: jQuery

Reply:
Alright, turns out this was because the method didn't check if a page was loaded before calling jQuery functions. I just fixed that.